backend/src/main/java/gov/cdc/usds/simplereport/service/model/UserIdentityInfo.java

```diff
@@ -0,0 +1,32 @@
+package gov.cdc.usds.simplereport.service.model;
+
+import gov.cdc.usds.simplereport.db.model.ApiUser;
+import gov.cdc.usds.simplereport.db.model.DatabaseEntity;
+import gov.cdc.usds.simplereport.db.model.PersonEntity;
+import gov.cdc.usds.simplereport.db.model.auxiliary.PersonName;
+import java.util.UUID;
+
+public class UserIdentityInfo extends WrappedEntity<ApiUser>
```

Comment:
This doesn't seem like a useful wrapper class to me—we could just return the ApiUser object and modify `ApiUserDataResolver` to extend `InternalIdResolver`, which would also conveniently allow the merger of schema types that I was suggesting elsewhere. 😸